tsweb/tsweb.go

```diff
@@ -137,6 +139,61 @@ func stripPort(hostport string) string {
 	return net.JoinHostPort(host, "443")
 }
 
+// Handler is like net/http.Handler, but the handler can return an
+// error.
+type Handler interface {
+	ServeHTTP(http.ResponseWriter, *http.Request) error
```

Comment:
I'd prefer a different name than ServeHTTP.

Also, say what the errors can be.